third_party/pyth/multisig-wh-message-builder/src/index.ts

```diff
@@ -361,7 +460,9 @@ async function getSquadsClient(
   ledger: boolean,
   ledgerDerivationAccount: number | undefined,
   ledgerDerivationChange: number | undefined,
-  walletPath: string
+  walletPath: string,
+  solRpcUrl?: string,
```

Comment:
I also feel like this should default to `http://localhost:8899` for localdevnet

Reply:
In Tilt you usually refer to the solana node by `solana-devnet`, I generally wanted to make interactions with arbitrary Solana deployments possible

Reply:
I'll make it default to the localhost:8899 thing and change it for tilt in the surrounding Python script